doge-webapp/build.gradle

```diff
@@ -1,23 +1,30 @@
 description = "doge webapp"
```

Comment:
I ended up hacking up this file a bit because I wanted to separate the Spring Boot changes from the Gradle Node Plugin changes. I also added the local resources to the `bootRun` configuration so that it supports "hot reloading."